### PR TITLE
feat: Bump Go version to 1.20

### PR DESCRIPTION
**Pull request by mbevc1**
* Bump Go to 1.20
* Update some dependencies

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -23,6 +23,6 @@ jobs:
     - name: Install Go
       uses: actions/setup-go@v3
       with:
-        go-version: 1.19
+        go-version: 1.20
```
[the remark this answers is not in the document]
Comment (mbevc1) [reply .github/workflows/build.yml:26]: Happy to put in in quotes, but never had issues using without :)

[the remark this answers is not in the document]
Comment (marcosnils) [reply .github/workflows/build.yml:26]: https://github.com/actions/setup-go/issues/328

did you go back to go 1.2? :sweat_smile:

[the remark this answers is not in the document]
Comment (mbevc1) [reply .github/workflows/build.yml:26]: Ah, I see now what you meant :smile:  Fixing... :hammer_and_wrench:

**Review by marcosnils — APPROVED** [review]

**Comment by mbevc1** [comment]
Cheers @marcosnils . It's probably worth cutting a fresh release build with 1.20? :thinking:

**Comment by marcosnils** [comment]
there's no rush since there's no functionality change withing the releases?

**Comment by mbevc1** [comment]
Yeah, not urgent but just build with newer Golang.

**Comment by marcosnils** [comment]
np. I was planning to fix some bugs the following days so I'll probably make the release then.

**Pull request merged**